<launch> 

    <rosparam command="load" file="$(find demo_pakage)/config/para.yaml" />

    <node pkg="demo_pakage" type="xml_test" name="lane_detection" output="screen"/>

</launch>
